charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -1,4 +1,4 @@
 java:
   applicationPort: 8583
   environment:
-    image: 'https://hmctspublic.azurecr.io/bulk-scan/pay-processor:latest'
+  image: 'https://hmctspublic.azurecr.io/bulk-scan/pay-processor:latest'
```

Comment:
perhaps image needs a full title too?

Reply:
I think image name should be fine but will update it 👍